src/authorizations/components/redesigned/IndividualAccordionBody.tsx

```diff
@@ -26,6 +26,15 @@ interface Props {
 
 export const IndividualAccordionBody: FC<Props> = props => {
   const {resourceName, permissions, onToggle, title, disabled} = props
+  let sortedPermissions
+
+  if (title === 'Individual Bucket Names') {
```

Comment:
is it possible to `resourceName === 'buckets'` here? I think it would be a less volatile property. A designer or UX expert may decide that `Individual Bucket Names` isn't what we want to call this section, and if we need to change the text and forget to update it here, we'd have a bug where this part of the app doesn't sort buckets consistently.

if it is possible to use `resourceName === 'buckets'`, you don't have to get a re-review with this change.

Reply:
gooood catch!! you're absolutely right. I was looking for a variable that would differentiate between buckets and telegrafs. I was looking else where when the answer was right under my nose